apps/nextjs-website/react-components/types/HeroChips/HeroChips.types.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonProps, Generic } from "../common/Common.types";
+
+export interface HeroChipsProps extends CommonProps {
+  readonly background?: string | Generic;
+  readonly title: string;
+  readonly subtitle?: string | Generic;
+  readonly isTitleCentered?: boolean;
+  readonly isSubtitleCentered?: boolean;
```

Comment:
```suggestion
  readonly centerText?: boolean;
```
I don't think it makes sense for these to be separate bools.
Text is either all center-aligned or it isn't.

Also, are chips always centered? If so, I'd make the text always centered as well.